### PR TITLE
Fintan

### PR DESCRIPTION
**Pull request by fguckian-11630129**
Pull request for changes made to FixBookCOntrol, FixBookUI, BorrowBookControl and BorrowBookUI.

### CODE REVIEW

File: src/Book.java [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 
 @SuppressWarnings("serial")
-public class book implements Serializable {
+public class Book implements Serializable {
 	
 	private String T;
```
Comment (KalyanManning) [inline src/Book.java]: T and A are not good variable names. Should be title and author respectively

File: src/Book.java [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 
 @SuppressWarnings("serial")
-public class book implements Serializable {
+public class Book implements Serializable {
```
Comment (KalyanManning) [inline src/Book.java]: T, A, C and ID are not good variable names: should be title, author, callNo and id respectively

File: src/BorrowBookControl.java [hunk 3]
```diff
@@ -3,74 +3,87 @@
 
 public class BorrowBookControl {
```
Comment (KalyanManning) [inline src/BorrowBookControl.java]: UI is not a good variable name: should be ui (all variable names should start with a lowercase letter, even abbreviations)

File: src/FixBookControl.java [hunk 4]
```diff
@@ -1,67 +1,79 @@
 public class FixBookControl {
```
Comment (KalyanManning) [inline src/FixBookControl.java]: Here again ui is preferable to UI

File: src/FixBookUI.java [hunk 5]
```diff
@@ -31,13 +31,13 @@ public void run() {
 			switch (state) {
```
Comment (KalyanManning) [inline src/FixBookUI.java]: Here input should be inputScanner

File: src/BorrowBookControl.java [hunk 6]
```diff
@@ -3,74 +3,87 @@
...
 	
-	private BorrowBookUI ui;
+	private BorrowBookUI UI;
```
Comment (KalyanManning) [inline src/BorrowBookControl.java]: Use ui instead of UI

File: src/FixBookControl.java [hunk 7]
```diff
@@ -1,67 +1,79 @@
...
 	
-	private FixBookUI ui;
+	//FixBookControl
+	private FixBookUI UI;
 	private enum CONTROL_STATE { INITIALISED, READY, FIXING };
```
Comment (KalyanManning) [inline src/FixBookControl.java]: Use ui instead of UI

File: src/BorrowBookControl.java (hunk 6 above)
Comment (fguckian-11630129) [reply src/BorrowBookControl.java]: Done

File: src/FixBookControl.java (hunk 7 above)
Comment (fguckian-11630129) [reply src/FixBookControl.java]: Done

**Pull request merged**